### DISCUSSION TITLE
Which kernel should I use if I want to treat time derivative term as 0 in CH equation?

### DISCUSSION THREAD

**Discussion by abc-hy** [idaholab/moose#21815] [Q&A Modules: Phase field] [answered]
Hi everyone,

Which kernel should I use if I don't want to consider time derivative term in CH equation?
I want to set time derivative term as 0.
Currently, I am using Coupledtimederivative, kernel to deal with the time derivative term, can I simply delete this part? 

Thank you,
Best

**Comment by GiudGiud** [comment, marked as answer]
So you want to remove the time derivative?
Then do not use any time derivative kernel

You may even switch to a Steady executioner
